specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -211,6 +211,14 @@ function TreeView<SCHEMA extends AnyTree>({
             if (!isHorizontal) resetDimensions();
           }}
         />
+        <Button.Icon
+          disabled={!isSplit}
+          icon="synchronize"
+          title={treeText.synchronize()}
+          onClick={() => {
+            states.second.focusPath[1](states.first.focusPath[0]);
```

Comment:
(optional)
instead of always setting 2nd tree to where 1st tree is, would it be better to set the least recently tree's focus to where the most recently used tree's focus is?

whichever tree is focused at the moment, can assume that it's the one that was used most recently

would that be better UX?

Reply:
mhhh might be... I just wanted to make it consistent. Always the same behavior, like that user understands directly wath is happening.